src/Core/Grid/Definition/Factory/Security/Session/EmployeeGridDefinitionFactory.php

```diff
@@ -0,0 +1,263 @@
+use PrestaShop\PrestaShop\Core\Grid\Definition\Factory\DeleteActionTrait;
+use PrestaShop\PrestaShop\Core\Grid\Filter\Filter;
+use PrestaShop\PrestaShop\Core\Grid\Filter\FilterCollection;
+use PrestaShop\PrestaShop\Core\Grid\Filter\FilterCollectionInterface;
+use PrestaShopBundle\Form\Admin\Type\DateRangeType;
+use PrestaShopBundle\Form\Admin\Type\SearchAndResetType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+
+/**
+ * Class EmployeeGridDefinitionFactory responsible for creating Employee grid definition.
+ */
+final class EmployeeGridDefinitionFactory extends AbstractGridDefinitionFactory
+{
+    use BulkDeleteActionTrait;
+    use DeleteActionTrait;
+
+    /**
+     * @var string
+     */
+    public const GRID_ID = 'security_session_employee';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getId(): string
+    {
+        return static::GRID_ID;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getName(): string
+    {
+        return $this->trans('Employee sessions', [], 'Admin.Navigation.Menu');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getColumns(): ColumnCollectionInterface
+    {
+        return (new ColumnCollection())
+            ->add(
+                (new BulkActionColumn('bulk'))
+                    ->setOptions([
+                        'bulk_field' => 'id_employee_session',
+                    ])
+            )
+            ->add(
+                (new DataColumn('id_employee_session'))
+                    ->setName($this->trans('ID', [], 'Admin.Global'))
+                    ->setOptions([
+                        'field' => 'id_employee_session',
+                    ])
+            )
+            ->add(
+                (new DataColumn('id_employee'))
+                    ->setName($this->trans('Employee ID', [], 'Admin.Global'))
+                    ->setOptions([
+                        'field' => 'id_employee',
+                    ])
+            )
+            ->add(
+                (new DataColumn('firstname'))
+                    ->setName($this->trans('First name', [], 'Admin.Global'))
+                    ->setOptions([
+                        'field' => 'firstname',
+                    ])
+            )
+            ->add(
+                (new DataColumn('lastname'))
+                    ->setName($this->trans('Last name', [], 'Admin.Global'))
+                    ->setOptions([
+                        'field' => 'lastname',
+                    ])
+            )
+            ->add(
+                (new DataColumn('email'))
+                    ->setName($this->trans('Email', [], 'Admin.Global'))
+                    ->setOptions([
+                        'field' => 'email',
+                    ])
+            )
+            ->add(
+                (new DataColumn('date_upd'))
+                    ->setName($this->trans('Updated at', [], 'Admin.Advparameters.Feature'))
```

Comment:
Same question here. If it is followed by a date, 'at' should be replaced by 'on'.